spec/factories.rb

```diff
@@ -503,8 +503,8 @@
 
   factory :poll_question_answer, class: 'Poll::Question::Answer' do
     association :question, factory: :poll_question
-    sequence(:title) { |n| "Question title #{n}" }
-    sequence(:description) { |n| "Question description #{n}" }
+    sequence(:title) { |n| "Answer title #{n}" }
+    sequence(:description) { |n| "Answer description #{n}" }
```

Comment:
I changed the name for the `poll_question_answer` because it made conflicts with the `poll_question` name when running specs (they had the same name, but, as I understand, the answers should be named 'Answer title X' instead of 'Question title X').